internal/controllers/routes.go

```diff
@@ -75,12 +76,12 @@ func (r *CryostatReconciler) reconcileGrafanaRoute(ctx context.Context, svc *cor
 			Namespace: cr.Namespace,
 		},
 	}
-
+	grafanaConfig := configureGrafanaRoute(cr)
```

Comment:
I would move this to after the following if-statement. This avoids calling this function if it's a minimal deployment.

Reply:
Oh yess true!